python/ml_croissant/ml_croissant/_src/core/json_ld.py

```diff
@@ -0,0 +1,150 @@
+"""json_ld module."""
```

Comment:
Maybe have a short description of what this module is used for instead.

Reply:
Done.